default/python/AI/fleet_orders.py

```diff
@@ -401,6 +404,7 @@ def is_valid(self):
         if not super(OrderInvade, self).is_valid():
             return False
         planet = self.target.get_object()
+        # TODO: Shouldn't this be initial meter value to be consistent with AIFleetMission checks?
```

Comment:
I am not finding an Invasion-related population meter check in AIFleetMission for this to be inconsistent with, but regardless (and this is seeming to me like I made nearly the same comment re invasions above, but cannot find it now, though it is a bit related to the colonization check)-- if this check were only being done at the very last instant, just before issuing the actual invade order, then I'd agree that initialMeter is the way to go.  Since I think it is often checked in advance, then it seems more logical to me to check currentMeter (though frankly, since this only matters for unowned planets, with current content I can't even see when those readings would give a different result),